inc/storefront-template-functions.php

```diff
@@ -845,8 +845,12 @@ function storefront_homepage_content() {
 		while ( have_posts() ) {
 			the_post();
 
-			get_template_part( 'content', 'homepage' );
+			$title   = get_the_title( get_the_ID() );
+			$content = get_the_content( get_the_ID() );
```

Comment:
Incorrect usage of `get_the_content`. https://codex.wordpress.org/Function_Reference/get_the_content